README.md

```diff
@@ -20,9 +20,12 @@ See [the docs](https://docs.rs/rodio/latest/rodio/#alternative-decoder-backends)
 
 [The documentation](http://docs.rs/rodio) contains an introduction to the library.
 
+# [Breaking Changes](UPGRADE.md)
+We have written an upgrade [guide](UPGRADE.md), it will help you upgrade to rodio 0.21.
+
 # [Examples](https://github.com/RustAudio/rodio/tree/f1eaaa4a6346933fc8a58d5fd1ace170946b3a94/examples)
 
-We are currently making large improvements to rodio. This does mean the updated examples do not work with the current crates.io release. You will have to look at the examples from commit `f1eaaa4a`. They are available [on github](https://github.com/RustAudio/rodio/tree/f1eaaa4a6346933fc8a58d5fd1ace170946b3a94/examples).
+We are always making changes to rodio. That can mean the examples do not work with the current crates.io release. You will have to look at the examples from commit `f1eaaa4a`. They are available [on github](https://github.com/RustAudio/rodio/tree/f1eaaa4a6346933fc8a58d5fd1ace170946b3a94/examples).
```

Comment:
1. Suggestion: "This can mean that..."
2. Why this specific permalink to the examples? Don't we always try to keep the CI passing, which means keeping the examples and tests in sync with current head?

Reply:
1. agreed
2. we had a report from a user who tried the repo examples on the crates-io release. An alternative could be moving all development to a dev branch. But that has issues with new contributors trying to merge to master instead of dev.